scripts/fieldbus/openfdd_bench_gate.sh

```
@@ -0,0 +1,34 @@
+# Runs the validation suite the way Open-FDD would use the sidecar in production:
+#   1. scripts/smoke_test.sh           (fast REST fail gate)
+#   2. scripts/bench_test.sh           (BACnet matrix + Modbus + Haystack + PCAP per phase)
+#
+# Usage:
+#   OPENFDD_FIELDBUS_API_KEY=... scripts/openfdd_bench_gate.sh
+#   BENCH_MINUTES=30 scripts/openfdd_bench_gate.sh   # half-hour soak inside bench_test
+set -euo pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+# shellcheck source=scripts/bench_lib.sh
+source "$ROOT/scripts/bench_lib.sh"
```

Shellcheck (0.11.0)

[info] Not following: scripts/bench_lib.sh was not specified as input (see shellcheck -x).

(SC1091)